terraform/helm/monitoring/values.yaml

```diff
@@ -2,9 +2,10 @@ chain:
   name:
 validator:
   name:
+  releaseName:
```

Comment:
what is this for?

Reply:
uhh another testing leftover...thanks for catching it